### ISSUE TITLE
Uninstall doesn't work, update doesn't work

### ISSUE THREAD

**Issue by 20PercentRendered** [Heroic-Games-Launcher/HeroicGamesLauncher#1505]
### Describe the bug

Trying to launch Rocket League prompts me with "An update is available" and pressing "no" launches the game, however Heroic doesn't show the game as running. Attempting to update the game updates it, but Heroic prompts immediately afterwards that "An update is available". 
Uninstalling a game after updating it doesn't do anything, Heroic says the game is still installed (fixed by restarting Heroic and pressing uninstall again). 

### Add logs

(12:40:20) DEBUG:   [Legendary]:       Running Legendary command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
(12:40:21) INFO:    [Legendary]:       Legendary location: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary
(12:40:21) INFO:    [Gog]:             GOGDL location: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl
(12:40:21) INFO:    [Backend]:         

Heroic Version: 2.3.9 Brook
Legendary Version:  0.20.26 Dark Energy
OS: Arch KERNEL: 5.18.5-zen1-1-zen ARCH: x64
CPU: Intel Core™ i7-7800X @3.5 GOVERNOR: performance
RAM: Total: 46.74 GiB Available: 40.88 GiB
GRAPHICS: GPU0: NVIDIA GeForce RTX 2080 VRAM: 8192MB DRIVER: 515.48.07 
PROTOCOL: x11

(12:40:21) WARNING: [Backend]:         Protocol already registered.
(12:40:22) INFO:    [Frontend]:        Refreshing Library
(12:40:22) INFO:    [Legendary]:       Refreshing library...
(12:40:22) INFO:    [Legendary]:       Refreshing Epic Games...
(12:40:22) DEBUG:   [Legendary]:       Running Legendary command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary list
(12:40:23) INFO:    [Backend]:         Downloaded Winetricks
(12:40:24) INFO:    [Legendary]:       Updating game list
(12:40:24) INFO:    [Legendary]:       Game List Updated
(12:40:25) INFO:    [Legendary]:       Checking for game updates: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary list-installed --check-updates --tsv
(12:40:25) DEBUG:   [Legendary]:       Running Legendary command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary list-installed --check-updates --tsv
(12:40:26) INFO:    [Legendary]:       Found 1 game(s) to update
(12:40:26) INFO:    [Gog]:             Found 0 game(s) to update
(12:40:36) INFO:    [Backend]:         Launching Dead by Daylight (Brill)
(12:40:36) WARNING: [Backend]:         You are using Proton, this can lead to some bugs. Please do not open issues with bugs related to games
(12:40:36) WARNING: [Legendary]:       No Steam runtime found
(12:40:36) INFO:    [Legendary]:       Launching Dead by Daylight: STEAM_COMPAT_CLIENT_INSTALL_PATH=/home/onni/.steam/steam STEAM_COMPAT_DATA_PATH=/mnt/deathclaw/EpicPrefixes/DeadByDaylight STEAM_COMPAT_APP_ID=0 SteamAppId=0 SteamGameId=heroic-DeadByDaylight PROTON_LOG_DIR=/home/onni /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary launch Brill --no-wine --wrapper "'/mnt/deathclaw/SteamLibrary/steamapps/common/Proton - Experimental/proton' run"
(12:40:36) DEBUG:   [Legendary]:       Running Legendary command: STEAM_COMPAT_CLIENT_INSTALL_PATH=/home/onni/.steam/steam STEAM_COMPAT_DATA_PATH=/mnt/deathclaw/EpicPrefixes/DeadByDaylight STEAM_COMPAT_APP_ID=0 SteamAppId=0 SteamGameId=heroic-DeadByDaylight PROTON_LOG_DIR=/home/onni /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary launch Brill --no-wine --wrapper "'/mnt/deathclaw/SteamLibrary/steamapps/common/Proton - Experimental/proton' run"
(12:41:56) DEBUG:   [Legendary]:       Running Legendary command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
(12:41:56) INFO:    [Backend]:         Log was written to /home/onni/.config/heroic/GamesConfig/Brill-lastPlay.log
(12:41:56) INFO:    [Frontend]:        Refreshing Library
(12:41:56) INFO:    [Legendary]:       Refreshing library...
(12:41:56) INFO:    [Legendary]:       Updating game list
(12:41:56) INFO:    [Legendary]:       Game List Updated
(12:42:22) DEBUG:   [Legendary]:       Using cached install info
(12:42:25) DEBUG:   [Legendary]:       Using cached install info
(12:42:25) DEBUG:   [Legendary]:       Using cached install info
(12:42:25) INFO:    [Legendary]:       Updating Sugar with: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary update Sugar -y
(12:42:25) DEBUG:   [Legendary]:       Running Legendary command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary update Sugar -y
(12:42:25) DEBUG:   [Legendary]:       Logging to file "/home/onni/.config/heroic/GamesConfig/Sugar.log"
(12:43:00) INFO:    [Legendary]:       Installed game list updated
(12:43:00) INFO:    [Legendary]:       Installed game list updated
(12:43:00) INFO:    [Backend]:         finished updating
(12:43:00) INFO:    [Frontend]:        Refreshing Library
(12:43:00) INFO:    [Legendary]:       Refreshing library...
(12:43:00) INFO:    [Legendary]:       Updating game list
(12:43:00) INFO:    [Legendary]:       Game List Updated
(12:43:00) INFO:    [Backend]:         operation interrupted
(12:43:00) DEBUG:   [Legendary]:       Using cached install info
(12:43:00) DEBUG:   [Legendary]:       Using cached install info
(12:43:00) DEBUG:   [Legendary]:       Using cached install info
(12:43:01) INFO:    [Legendary]:       Checking for game updates: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary list-installed --check-updates --tsv
(12:43:01) DEBUG:   [Legendary]:       Running Legendary command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary list-installed --check-updates --tsv
(12:43:02) INFO:    [Legendary]:       Found 1 game(s) to update
(12:43:02) DEBUG:   [Legendary]:       Using cached install info
(12:43:16) DEBUG:   [Legendary]:       Using cached install info
(12:43:18) INFO:    [Backend]:         Launching Rocket League® (Sugar)
(12:43:18) WARNING: [Backend]:         You are using Proton, this can lead to some bugs. Please do not open issues with bugs related to games
(12:43:18) DEBUG:   [Legendary]:       Running Wine command: "/mnt/deathclaw/SteamLibrary/steamapps/common/Proton - Experimental/proton" run wineboot --init
(12:43:21) DEBUG:   [Legendary]:       Ran Wine command: "/mnt/deathclaw/SteamLibrary/steamapps/common/Proton - Experimental/proton" run wineboot --init
(12:43:21) INFO:    [Backend]:         Created/Updated Wineprefix at /home/onni/Games/Heroic/Prefixes/rocketleague
(12:43:21) WARNING: [Legendary]:       No Steam runtime found
(12:43:21) INFO:    [Legendary]:       Launching Rocket League®: STEAM_COMPAT_CLIENT_INSTALL_PATH=/home/onni/.steam/steam STEAM_COMPAT_DATA_PATH=/home/onni/Games/Heroic/Prefixes/rocketleague STEAM_COMPAT_APP_ID=0 SteamAppId=0 SteamGameId=heroic-rocketleague PROTON_LOG_DIR=/home/onni /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary launch Sugar --no-wine --wrapper "'/mnt/deathclaw/SteamLibrary/steamapps/common/Proton - Experimental/proton' run" --skip-version-check
(12:43:21) DEBUG:   [Legendary]:       Running Legendary command: STEAM_COMPAT_CLIENT_INSTALL_PATH=/home/onni/.steam/steam STEAM_COMPAT_DATA_PATH=/home/onni/Games/Heroic/Prefixes/rocketleague STEAM_COMPAT_APP_ID=0 SteamAppId=0 SteamGameId=heroic-rocketleague PROTON_LOG_DIR=/home/onni /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary launch Sugar --no-wine --wrapper "'/mnt/deathclaw/SteamLibrary/steamapps/common/Proton - Experimental/proton' run" --skip-version-check
(12:43:24) INFO:    [Backend]:         Launching Rocket League® (Sugar)
(12:43:24) WARNING: [Backend]:         You are using Proton, this can lead to some bugs. Please do not open issues with bugs related to games
(12:43:24) WARNING: [Legendary]:       No Steam runtime found
(12:43:24) INFO:    [Legendary]:       Launching Rocket League®: STEAM_COMPAT_CLIENT_INSTALL_PATH=/home/onni/.steam/steam STEAM_COMPAT_DATA_PATH=/home/onni/Games/Heroic/Prefixes/rocketleague STEAM_COMPAT_APP_ID=0 SteamAppId=0 SteamGameId=heroic-rocketleague PROTON_LOG_DIR=/home/onni /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary launch Sugar --no-wine --wrapper "'/mnt/deathclaw/SteamLibrary/steamapps/common/Proton - Experimental/proton' run" --skip-version-check
(12:43:24) DEBUG:   [Legendary]:       Running Legendary command: STEAM_COMPAT_CLIENT_INSTALL_PATH=/home/onni/.steam/steam STEAM_COMPAT_DATA_PATH=/home/onni/Games/Heroic/Prefixes/rocketleague STEAM_COMPAT_APP_ID=0 SteamAppId=0 SteamGameId=heroic-rocketleague PROTON_LOG_DIR=/home/onni /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary launch Sugar --no-wine --wrapper "'/mnt/deathclaw/SteamLibrary/steamapps/common/Proton - Experimental/proton' run" --skip-version-check
(12:43:26) DEBUG:   [Legendary]:       Using cached install info
(12:43:26) INFO:    [Legendary]:       Updating Sugar with: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary update Sugar -y
(12:43:26) DEBUG:   [Legendary]:       Running Legendary command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary update Sugar -y
(12:43:26) DEBUG:   [Legendary]:       Logging to file "/home/onni/.config/heroic/GamesConfig/Sugar.log"
(12:43:27) DEBUG:   [Legendary]:       Using cached install info
(12:43:27) INFO:    [Legendary]:       Updating Sugar with: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary update Sugar -y
(12:43:27) DEBUG:   [Legendary]:       Running Legendary command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary update Sugar -y
(12:43:27) DEBUG:   [Legendary]:       Logging to file "/home/onni/.config/heroic/GamesConfig/Sugar.log"
(12:43:30) INFO:    [Backend]:         Progress for Sugar: 91.21%/0.00MiB/00:00:00
(12:43:31) INFO:    [Backend]:         Progress for Sugar: 91.35%/28.14MiB/00:03:02
(12:43:31) INFO:    [Backend]:         Progress for Sugar: 91.29%/15.00MiB/00:04:26
(12:43:32) INFO:    [Backend]:         Progress for Sugar: 91.53%/62.13MiB/00:05:20
(12:43:32) INFO:    [Backend]:         Progress for Sugar: 91.36%/29.14MiB/00:06:04
(12:43:33) INFO:    [Backend]:         Progress for Sugar: 91.66%/86.13MiB/00:07:03
(12:43:33) INFO:    [Backend]:         Progress for Sugar: 91.46%/49.13MiB/00:08:15
(12:43:34) INFO:    [Backend]:         Progress for Sugar: 91.81%/114.13MiB/00:08:29
(12:43:34) INFO:    [Backend]:         Progress for Sugar: 91.56%/67.13MiB/00:10:03
(12:43:35) INFO:    [Backend]:         Progress for Sugar: 91.96%/143.12MiB/00:09:44
(12:43:35) INFO:    [Backend]:         Progress for Sugar: 91.69%/92.13MiB/00:11:35
(12:43:36) INFO:    [Backend]:         Progress for Sugar: 92.14%/177.12MiB/00:10:37
(12:43:36) INFO:    [Backend]:         Progress for Sugar: 91.84%/121.13MiB/00:12:29
(12:43:37) INFO:    [Backend]:         Progress for Sugar: 92.34%/216.11MiB/00:11:34
(12:43:37) INFO:    [Backend]:         Progress for Sugar: 92.01%/152.12MiB/00:13:16
(12:43:38) INFO:    [Backend]:         Progress for Sugar: 92.48%/242.09MiB/00:11:43
(12:43:38) INFO:    [Backend]:         Progress for Sugar: 92.14%/178.12MiB/00:14:04
(12:43:39) INFO:    [Backend]:         Progress for Sugar: 92.64%/272.09MiB/00:12:15
(12:43:39) INFO:    [Backend]:         Progress for Sugar: 92.3%/207.12MiB/00:14:25
(12:43:40) INFO:    [Backend]:         Progress for Sugar: 92.74%/292.20MiB/00:10:52
(12:43:40) INFO:    [Backend]:         Progress for Sugar: 92.46%/239.09MiB/00:14:43
(12:43:41) INFO:    [Backend]:         Progress for Sugar: 92.84%/311.38MiB/00:07:08
(12:43:41) INFO:    [Backend]:         Progress for Sugar: 92.64%/272.09MiB/00:15:14
(12:43:42) INFO:    [Backend]:         Progress for Sugar: 93%/341.67MiB/00:04:26
(12:43:42) INFO:    [Backend]:         Progress for Sugar: 92.82%/305.81MiB/00:09:00
(12:43:43) INFO:    [Backend]:         Progress for Sugar: 93.21%/380.61MiB/00:03:54
(12:43:43) INFO:    [Backend]:         Progress for Sugar: 92.98%/336.71MiB/00:05:07
(12:43:44) INFO:    [Backend]:         Progress for Sugar: 93.39%/415.06MiB/00:04:03
(12:43:44) INFO:    [Backend]:         Progress for Sugar: 93.18%/375.68MiB/00:04:04
(12:43:45) INFO:    [Backend]:         Progress for Sugar: 93.55%/445.46MiB/00:03:57
(12:43:45) INFO:    [Backend]:         Progress for Sugar: 93.35%/407.45MiB/00:04:06
(12:43:46) INFO:    [Backend]:         Progress for Sugar: 93.73%/479.63MiB/00:03:24
(12:43:46) INFO:    [Backend]:         Progress for Sugar: 93.56%/446.53MiB/00:03:44
(12:43:47) INFO:    [Backend]:         Progress for Sugar: 93.87%/507.10MiB/00:02:51
(12:43:47) INFO:    [Backend]:         Progress for Sugar: 93.78%/488.61MiB/00:03:25
(12:43:48) INFO:    [Backend]:         Progress for Sugar: 94.01%/532.71MiB/00:02:21
(12:43:48) INFO:    [Backend]:         Progress for Sugar: 93.95%/520.68MiB/00:02:18
(12:43:49) INFO:    [Backend]:         Progress for Sugar: 94.14%/558.44MiB/00:02:16
(12:43:49) INFO:    [Backend]:         Progress for Sugar: 94.17%/563.89MiB/00:02:11
(12:43:50) INFO:    [Backend]:         Progress for Sugar: 94.31%/590.23MiB/00:02:00
(12:43:50) INFO:    [Backend]:         Progress for Sugar: 94.34%/595.59MiB/00:01:52
(12:43:51) INFO:    [Backend]:         Progress for Sugar: 94.45%/616.59MiB/00:01:52
(12:43:51) INFO:    [Backend]:         Progress for Sugar: 94.54%/633.45MiB/00:01:43
(12:43:52) INFO:    [Backend]:         Progress for Sugar: 94.62%/649.06MiB/00:01:48
(12:43:52) INFO:    [Backend]:         Progress for Sugar: 94.76%/675.88MiB/00:01:41
(12:43:53) INFO:    [Backend]:         Progress for Sugar: 94.79%/680.70MiB/00:01:49
(12:43:53) INFO:    [Backend]:         Progress for Sugar: 94.95%/710.72MiB/00:01:34
(12:43:54) INFO:    [Backend]:         Progress for Sugar: 94.94%/709.59MiB/00:01:42
(12:43:54) INFO:    [Backend]:         Progress for Sugar: 95.13%/745.84MiB/00:01:25
(12:43:55) INFO:    [Backend]:         Progress for Sugar: 95.09%/738.30MiB/00:01:31
(12:43:55) INFO:    [Backend]:         Progress for Sugar: 95.36%/788.62MiB/00:01:25
(12:43:56) INFO:    [Backend]:         Progress for Sugar: 95.27%/771.53MiB/00:01:31
(12:43:56) INFO:    [Backend]:         Progress for Sugar: 95.52%/820.15MiB/00:01:23
(12:43:57) INFO:    [Backend]:         Progress for Sugar: 95.38%/793.61MiB/00:01:32
(12:43:57) INFO:    [Backend]:         Progress for Sugar: 95.68%/850.75MiB/00:01:18
(12:43:58) DEBUG:   [Legendary]:       Running Legendary command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
(12:43:58) INFO:    [Backend]:         Progress for Sugar: 95.57%/828.40MiB/00:01:28
(12:43:58) INFO:    [Backend]:         Progress for Sugar: 95.89%/890.29MiB/00:01:11
(12:43:59) DEBUG:   [Legendary]:       Running Legendary command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
(12:43:59) INFO:    [Backend]:         Log was written to /home/onni/.config/heroic/GamesConfig/Sugar-lastPlay.log
(12:43:59) INFO:    [Backend]:         Log was written to /home/onni/.config/heroic/GamesConfig/Sugar-lastPlay.log
(12:43:59) INFO:    [Backend]:         Progress for Sugar: 95.72%/857.91MiB/00:01:25
(12:43:59) INFO:    [Backend]:         Progress for Sugar: 96.1%/929.84MiB/00:01:07
(12:44:00) INFO:    [Backend]:         Progress for Sugar: 95.89%/889.44MiB/00:01:20
(12:44:01) INFO:    [Backend]:         Progress for Sugar: 96.3%/967.67MiB/00:01:09
(12:44:01) INFO:    [Backend]:         Progress for Sugar: 96.03%/916.46MiB/00:01:18
(12:44:01) INFO:    [Backend]:         Launching Rocket League® (Sugar)
(12:44:01) WARNING: [Backend]:         You are using Proton, this can lead to some bugs. Please do not open issues with bugs related to games
(12:44:01) WARNING: [Legendary]:       No Steam runtime found
(12:44:01) INFO:    [Legendary]:       Launching Rocket League®: STEAM_COMPAT_CLIENT_INSTALL_PATH=/home/onni/.steam/steam STEAM_COMPAT_DATA_PATH=/home/onni/Games/Heroic/Prefixes/rocketleague STEAM_COMPAT_APP_ID=0 SteamAppId=0 SteamGameId=heroic-rocketleague PROTON_LOG_DIR=/home/onni /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary launch Sugar --no-wine --wrapper "'/mnt/deathclaw/SteamLibrary/steamapps/common/Proton - Experimental/proton' run" --skip-version-check
(12:44:01) DEBUG:   [Legendary]:       Running Legendary command: STEAM_COMPAT_CLIENT_INSTALL_PATH=/home/onni/.steam/steam STEAM_COMPAT_DATA_PATH=/home/onni/Games/Heroic/Prefixes/rocketleague STEAM_COMPAT_APP_ID=0 SteamAppId=0 SteamGameId=heroic-rocketleague PROTON_LOG_DIR=/home/onni /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary launch Sugar --no-wine --wrapper "'/mnt/deathclaw/SteamLibrary/steamapps/common/Proton - Experimental/proton' run" --skip-version-check
(12:44:02) INFO:    [Backend]:         Progress for Sugar: 96.51%/1007.09MiB/00:01:11
(12:44:02) INFO:    [Backend]:         Progress for Sugar: 96.17%/944.18MiB/00:01:14
(12:44:03) INFO:    [Backend]:         Progress for Sugar: 96.67%/1037.88MiB/00:01:13
(12:44:03) INFO:    [Backend]:         Progress for Sugar: 96.36%/980.23MiB/00:01:16
(12:44:04) INFO:    [Backend]:         Progress for Sugar: 96.86%/1074.84MiB/00:01:02
(12:44:04) INFO:    [Backend]:         Progress for Sugar: 96.54%/1014.16MiB/00:01:18
(12:44:05) INFO:    [Backend]:         Progress for Sugar: 97.01%/1102.40MiB/00:00:53
(12:44:05) INFO:    [Backend]:         Progress for Sugar: 96.7%/1044.36MiB/00:01:20
(12:44:06) INFO:    [Backend]:         Progress for Sugar: 97.19%/1137.98MiB/00:00:47
(12:44:06) DEBUG:   [Legendary]:       Running Legendary command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
(12:44:06) INFO:    [Backend]:         Log was written to /home/onni/.config/heroic/GamesConfig/Sugar-lastPlay.log
(12:44:06) INFO:    [Backend]:         Progress for Sugar: 96.9%/1081.95MiB/00:01:14
(12:44:07) INFO:    [Backend]:         Progress for Sugar: 97.36%/1169.21MiB/00:00:39
(12:44:07) INFO:    [Backend]:         Progress for Sugar: 97.08%/1116.44MiB/00:01:00
(12:44:08) INFO:    [Backend]:         Progress for Sugar: 97.56%/1206.66MiB/00:00:33
(12:44:08) INFO:    [Backend]:         Progress for Sugar: 97.21%/1140.91MiB/00:00:53
(12:44:09) INFO:    [Backend]:         Progress for Sugar: 97.73%/1239.36MiB/00:00:28
(12:44:10) INFO:    [Backend]:         Progress for Sugar: 97.91%/1273.91MiB/00:00:25
(12:44:10) INFO:    [Backend]:         Progress for Sugar: 97.52%/1200.80MiB/00:00:39
(12:44:12) INFO:    [Backend]:         Progress for Sugar: 98.26%/1341.02MiB/00:00:21
(12:44:12) INFO:    [Backend]:         Progress for Sugar: 97.73%/1239.35MiB/00:00:31
(12:44:13) INFO:    [Backend]:         Progress for Sugar: 98.33%/1353.84MiB/00:00:18
(12:44:13) INFO:    [Backend]:         Progress for Sugar: 97.89%/1269.96MiB/00:00:28
(12:44:14) INFO:    [Backend]:         Progress for Sugar: 98.5%/1386.50MiB/00:00:18
(12:44:14) INFO:    [Backend]:         Progress for Sugar: 98.03%/1296.59MiB/00:00:28
(12:44:15) INFO:    [Backend]:         Progress for Sugar: 98.63%/1411.20MiB/00:00:17
(12:44:15) INFO:    [Backend]:         Progress for Sugar: 98.2%/1328.20MiB/00:00:24
(12:44:16) INFO:    [Backend]:         Progress for Sugar: 98.83%/1449.76MiB/00:00:14
(12:44:16) INFO:    [Backend]:         Progress for Sugar: 98.38%/1363.76MiB/00:00:21
(12:44:17) INFO:    [Backend]:         Progress for Sugar: 99%/1481.39MiB/00:00:13
(12:44:17) INFO:    [Backend]:         Progress for Sugar: 98.55%/1395.41MiB/00:00:20
(12:44:18) INFO:    [Backend]:         Progress for Sugar: 99.15%/1508.94MiB/00:00:12
(12:44:18) INFO:    [Backend]:         Progress for Sugar: 98.76%/1435.92MiB/00:00:21
(12:44:19) INFO:    [Backend]:         Progress for Sugar: 99.33%/1543.77MiB/00:00:12
(12:44:19) INFO:    [Backend]:         Progress for Sugar: 98.91%/1463.59MiB/00:00:21
(12:44:20) INFO:    [Backend]:         Progress for Sugar: 99.46%/1568.23MiB/00:00:12
(12:44:20) INFO:    [Backend]:         Progress for Sugar: 99.02%/1485.31MiB/00:00:22
(12:44:21) INFO:    [Backend]:         Progress for Sugar: 99.58%/1590.46MiB/00:00:13
(12:44:21) INFO:    [Backend]:         Progress for Sugar: 99.16%/1510.93MiB/00:00:22
(12:44:22) INFO:    [Backend]:         Progress for Sugar: 99.73%/1619.64MiB/00:00:13
(12:44:22) INFO:    [Backend]:         Progress for Sugar: 99.34%/1545.76MiB/00:00:22
(12:44:23) INFO:    [Backend]:         Progress for Sugar: 99.91%/1653.81MiB/00:00:13
(12:44:23) DEBUG:   [Legendary]:       Using cached install info
(12:44:23) INFO:    [Legendary]:       Updating Sugar with: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary update Sugar -y
(12:44:23) DEBUG:   [Legendary]:       Running Legendary command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary update Sugar -y
(12:44:23) DEBUG:   [Legendary]:       Logging to file "/home/onni/.config/heroic/GamesConfig/Sugar.log"
(12:44:23) INFO:    [Backend]:         Progress for Sugar: 99.54%/1582.92MiB/00:00:21
(12:44:24) INFO:    [Backend]:         Progress for Sugar: 100%/1671.17MiB/00:00:11
(12:44:24) INFO:    [Backend]:         Progress for Sugar: 99.78%/1629.70MiB/00:00:18
(12:44:25) INFO:    [Backend]:         Progress for Sugar: 100%/1671.17MiB/00:00:09
(12:44:26) INFO:    [Backend]:         Progress for Sugar: 100%/1671.17MiB/00:00:06
(12:44:26) INFO:    [Backend]:         Progress for Sugar: 98.21%/0.00MiB/00:00:00
(12:44:26) INFO:    [Legendary]:       Uninstalling Sugar: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary uninstall Sugar -y
(12:44:26) DEBUG:   [Legendary]:       Running Legendary command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary uninstall Sugar -y
(12:44:27) INFO:    [Backend]:         Progress for Sugar: 107%/1671.17MiB/00:00:04
(12:44:27) INFO:    [Backend]:         Progress for Sugar: 98.39%/35.26MiB/00:00:14
(12:44:27) INFO:    [Backend]:         Progress for Sugar: 107%/1671.17MiB/00:00:13
(12:44:28) INFO:    [Backend]:         Progress for Sugar: 107%/1671.17MiB/00:00:00
(12:44:28) INFO:    [Backend]:         Progress for Sugar: 98.77%/105.99MiB/00:00:14
(12:44:29) INFO:    [Backend]:         Progress for Sugar: 107%/1671.17MiB/00:00:10
(12:44:29) INFO:    [Backend]:         Progress for Sugar: 107%/1671.17MiB/00:00:00
(12:44:29) INFO:    [Backend]:         Progress for Sugar: 99.13%/175.32MiB/00:00:19
(12:44:29) INFO:    [Legendary]:       Installed game list updated
(12:44:29) ERROR:   [Legendary]:       Corrupted intalled.json file, cannot load installed games SyntaxError: Unexpected end of JSON input
(12:44:29) INFO:    [Legendary]:       Installed game list updated
(12:44:29) INFO:    [Backend]:         finished uninstalling
(12:44:29) INFO:    [Backend]:         Desktop shortcut removed
(12:44:29) INFO:    [Backend]:         Applications shortcut removed
(12:44:30) INFO:    [Backend]:         Progress for Sugar: 107%/1671.17MiB/00:00:06
(12:44:30) INFO:    [Legendary]:       Installed game list updated
(12:44:30) ERROR:   [Legendary]:       Corrupted intalled.json file, cannot load installed games SyntaxError: Unexpected end of JSON input
(12:44:30) INFO:    [Legendary]:       Installed game list updated
(12:44:30) INFO:    [Backend]:         finished updating
(12:44:30) INFO:    [Backend]:         operation interrupted
(12:44:30) INFO:    [Backend]:         Progress for Sugar: 99.51%/247.51MiB/00:00:23
(12:44:31) INFO:    [Backend]:         Progress for Sugar: 107%/1671.17MiB/00:00:06
(12:44:31) INFO:    [Backend]:         Progress for Sugar: 99.86%/312.81MiB/00:00:29
(12:44:32) INFO:    [Backend]:         Progress for Sugar: 107%/1671.17MiB/00:00:06
(12:44:33) INFO:    [Backend]:         Progress for Sugar: 107%/1671.17MiB/00:00:07
(12:44:33) INFO:    [Backend]:         Progress for Sugar: 100%/340.34MiB/00:00:00

### Steps to reproduce

1. Open Heroic
2. Click on an installed game
3. Click update 
4. After update finishes, wait a couple seconds
5. "Update available" should appear again.
6. Go to the Library screen.
7. Right-click the game and click uninstall
8. You should get a notification that says "Uninstalled"
9. The game is still installed, but cannot be launched.

### Expected behavior

When clicking uninstall, the game should uninstall and remove itself from the list of installed games. 
After updating, there shouldn't be another update immediately afterward. 

### Screenshots

# "Played Recently" bug: 
![image](https://user-images.githubusercontent.com/32398752/174772477-d03c6371-ba2a-4dab-b2f7-32806a1bb2a8.png)
![image](https://user-images.githubusercontent.com/32398752/174772496-3983d58e-6164-46e4-81a4-43ca84de8b2a.png)

### System Information

- OS [e. g. "Ubuntu"]: Arch Linux 
- Heroic Version [e. g. 2.1.1]: 2.3.9

### Additional information

Games that get uninstalled properly will have their state stick in "Played Recently" category, but not in the "All Games" category.

**Comment by TabulateJarl8** [comment]
I get the updating issue with Train Sim World 2. Here's my log:
```
(21:04:54) DEBUG:   [Legendary]:       Running Legendary command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
(21:04:55) INFO:    [Legendary]:       Legendary location: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary
(21:04:55) INFO:    [Gog]:             GOGDL location: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl
(21:04:55) INFO:    [Backend]:         

Heroic Version: 2.3.9 Brook
Legendary Version:  0.20.26 Dark Energy
OS: Arch KERNEL: 5.18.7-arch1-1 ARCH: x64
CPU: AMD Ryzen 5 3600 6-Core Processor @3.6 GOVERNOR: schedutil
RAM: Total: 31.3 GiB Available: 28.1 GiB
GRAPHICS: GPU0: NVIDIA GeForce RTX 2070 SUPER VRAM: 8192MB DRIVER: 515.48.07 
PROTOCOL: x11

(21:04:55) WARNING: [Backend]:         Protocol already registered.
(21:04:56) INFO:    [Frontend]:        Refreshing Library
(21:04:56) INFO:    [Legendary]:       Refreshing library...
(21:04:56) INFO:    [Legendary]:       Refreshing Epic Games...
(21:04:56) DEBUG:   [Legendary]:       Running Legendary command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary list
(21:04:57) INFO:    [Legendary]:       Updating game list
(21:04:57) INFO:    [Legendary]:       Game List Updated
(21:04:57) INFO:    [Legendary]:       Checking for game updates: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary list-installed --check-updates --tsv
(21:04:57) DEBUG:   [Legendary]:       Running Legendary command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary list-installed --check-updates --tsv
(21:04:57) INFO:    [Legendary]:       Found 2 game(s) to update
(21:04:57) INFO:    [Gog]:             Found 0 game(s) to update
(21:04:58) INFO:    [Backend]:         Downloaded Winetricks
(21:04:58) DEBUG:   [Legendary]:       Using cached install info
(21:05:00) DEBUG:   [Legendary]:       Using cached install info
(21:05:00) DEBUG:   [Legendary]:       Using cached install info
(21:05:00) INFO:    [Legendary]:       Updating dd75b5c168d546cc869582c0389f0a1d with: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary update dd75b5c168d546cc869582c0389f0a1d -y
(21:05:00) DEBUG:   [Legendary]:       Running Legendary command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary update dd75b5c168d546cc869582c0389f0a1d -y
(21:05:00) DEBUG:   [Legendary]:       Logging to file "/home/tabulate/.config/heroic/GamesConfig/dd75b5c168d546cc869582c0389f0a1d.log"
(21:05:01) INFO:    [Backend]:         finished updating
(21:05:01) INFO:    [Frontend]:        Refreshing Library
(21:05:01) INFO:    [Legendary]:       Refreshing library...
(21:05:01) INFO:    [Legendary]:       Updating game list
(21:05:01) INFO:    [Legendary]:       Game List Updated
(21:05:01) INFO:    [Backend]:         operation interrupted
(21:05:01) DEBUG:   [Legendary]:       Using cached install info
(21:05:01) DEBUG:   [Legendary]:       Using cached install info
(21:05:01) DEBUG:   [Legendary]:       Using cached install info
(21:05:02) INFO:    [Legendary]:       Checking for game updates: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary list-installed --check-updates --tsv
(21:05:02) DEBUG:   [Legendary]:       Running Legendary command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary list-installed --check-updates --tsv
(21:05:02) INFO:    [Legendary]:       Found 2 game(s) to update
(21:05:02) DEBUG:   [Legendary]:       Using cached install info

```

**Comment by flavioislima** [comment]
On Beta 2 you can use the Force Update Feature now if needed. It is on Game Page > tools.
That should fix these cases.
